projects/spectator/src/lib/spectator/spectator.ts

```diff
@@ -45,4 +45,79 @@ export class Spectator<C> extends DomSpectator<C> {
     // Force cd on the host component for cases such as: https://github.com/ngneat/spectator/issues/539
     this.detectChanges();
   }
+
+  public get deferBlocks(): DeferBlocks {
+    return this._deferBlocksForGivenFixture(this.fixture.getDeferBlocks());
+  }
+
+  /**
+   *
+   * @param deferBlockFixture Defer block fixture
+   * @returns deferBlock object with methods to access the defer blocks
+   */
+  private _deferBlocksForGivenFixture(deferBlockFixture: Promise<DeferBlockFixture[]>): DeferBlocks {
+    return {
+      renderComplete: async (deferBlockIndex = 0) => {
```

Comment:
I think it makes more sense to call `spectator.deferBlock(optionalIndex).renderComplete()`, etc.

Reply:
Yes, this makes sense. 
It could be more readable to switch between the states 

```ts
const firstDeferBlock = spectator.deferBlock()
firstDeferBlock.renderComplete()
// assert
firstDeferBlock.renderPlaceholder()
// assert
```

I'll convert the getter to a method